src/client/src/Stream.c

```diff
@@ -229,6 +229,7 @@ STATUS createStream(PKinesisVideoClient pKinesisVideoClient, PStreamInfo pStream
                                  pKinesisVideoStream->streamInfo.streamCaps.bufferDuration,
                                  viewItemRemoved,
                                  TO_CUSTOM_DATA(pKinesisVideoStream),
+                                 DROP_UNTIL_FRAGMENT_START,
```

Comment:
I will take care of exposing this further in V2 of StreamCaps as I am doing the next step of the work on storage pressure